core/common/src/main/java/io/quarkus/artemis/core/common/ArtemisUtil.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.artemis.core.common;
+
+public class ArtemisUtil {
+    public static final String DEFAULT_CONFIG_NAME = "<default>";
+    public static final String DEFAULT_CONFIG_URL = "tcp://localhost:61616";
```

Comment:
No config url should mean use devservices

Reply:
Done